cmp/report_slices.go

```diff
@@ -91,8 +91,9 @@ func (opts formatOptions) FormatDiffSlice(v *valueNode) textNode {
 			if r == '\n' {
 				if maxLineLen < i-lastLineIdx {
 					lastLineIdx = i - lastLineIdx
```

Comment:
I think this logic was trying to update `maxLineLen`.

Thus, it should be:
```go
if r == '\n' {
	if maxLineLen < i-lastLineIdx {
		maxLineLen = i - lastLineIdx
	}
	lastLineIdx = i + 1
	numLines++
}
```

Reply:
That actually makes more sense, I guess. I'm happy to update the PR!

Reply:
That'd be great. You may need to update a few of the tests.

Reply:
Done!